packages/blade/src/components/ActionList/docs/stories.ts

```diff
@@ -83,15 +83,23 @@ const ActionListItem = `
     ActionListItem, 
     ActionListItemIcon, 
     ActionListItemText, 
+    ActionListItemBadgeGroup,
+    ActionListItemBadge,
     HomeIcon 
   } from '@razorpay/blade/components';
 
   function App(): React.ReactElement {
     return (
-    <Box>
+    <Box backgroundColor="surface.background.level2.lowContrast">
       <ActionList>
         <ActionListItem 
           title="Title"
+          titleSuffix={(
+            <ActionListItemBadgeGroup>
```

Comment:
can we make this example more contextual with the use case where this was raised? something with `in: something` IIRC?

Reply:
This one is just individual action list item example to show all capabilities so it has to be vague cause there is no context around.
<img width="1093" alt="image" src="https://github.com/razorpay/blade/assets/30949385/2e8820f0-07bf-464b-a1a1-2d4d62ed420a">

Added more contextual example here - https://github.com/razorpay/blade/pull/1639#discussion_r1328477137